lib/messagebird/callflow.rb

```diff
@@ -26,7 +26,8 @@ def updated_at=(value)
   end
 
   class CallFlowList < List
-    attr_accessor :perPage, :currentPage, :pageCount, :totalCount
```

Comment:
Aren't these breaking changes?

Reply:
I have a feeling that this is not going to work :(  

Because the API returns attributes in camelCase we will fail to "parse" them with these new underscored attributes

https://github.com/messagebird/ruby-rest-api/blob/bb63d1f1f87200e70949f80c6fac8ebf99f68780/lib/messagebird/callflow.rb#L33

Reply:
I tried to catch the problem with a test and failed 😂
https://github.com/messagebird/ruby-rest-api/pull/75/commits/bea7298de8978554062d7cc952ca0056fd381854

Reply:
You didn't catch the problem because there's no problem 😄 

When an `CallFlowList` is instantiated, the function `map_hash_elements_to_self` (`MessageBird::Base`) is executed (by inheritance: `CallFlowList` < `List` < `MessageBird::Base`). This function is responsible for converting camelCase JSON fields into snake_case object properties.